ports/esp32/Makefile

```diff
@@ -136,6 +136,7 @@ SRC_C = \
 	machine_uart.c \
 	modmachine.c \
 	modnetwork.c \
+        network_lan.c \
```

Comment:
Is this a tab/space inconsistency?

Reply:
Yes, looked weird in _meld_, so I "fixed" it.